docs/modules/Decode.ts.md

```diff
@@ -96,6 +108,21 @@ export const right: <A>(a: A) => Decoder<A> = ...
 
 Added in v0.5.0
 
+# fromType (function)
+
+Creates a `Decoder` from an `io-ts` type.
+
+**Signature**
+
+```ts
+export const fromType: <A>(t: Type<A, any, Mixed>) => Decoder<A> = type => v =>
```

Comment:
If `fromType` is reintroduced, maybe the input requirement should be lowered to io-ts's `Decoder`

Reply:
No, this was my fault: I didn't know that Giulio removed `fromType` because he wanted to remove `io-ts` as dependency (which is still reported as _peer_).

Some examples used that function and I reintroduced it in order to fix them.